tools/generator/src/DTO.swift

```diff
@@ -41,7 +41,7 @@ struct Product: Equatable, Decodable {
     let path: FilePath
 }
 
-struct Platform: Equatable, Decodable {
+struct Platform: Equatable, Hashable, Decodable {
```

Comment:
Is the hashable functionality for Platform used in this PR?

Reply:
Yes:
```
struct BuildSettingConditional: Equatable, Hashable {
     private let platform: Platform?
}
```